data/heb0001/heb010/__cts__.xml

```diff
@@ -6,6 +6,7 @@
     </cpt:structured-metadata>
   <ti:translation xml:lang= "eng" urn="urn:cts:hebrewlit:heb0001.heb010.1st1K-eng1" workUrn="urn:cts:hebrewlit:heb0001.heb010">
     <ti:label xml:lang="lat">Isaias</ti:label>
```

Comment:
I think the label would be `Isiah (from the Hebrew)` as that is the work title in print.

Reply:
I will change that!